colorcoder/Main.java

```diff
@@ -1,120 +1,29 @@
 package colorcoder;
 
```

Comment:
in Main.java class name and "main" method name is same ....what is the significance of same name?